app/assets/stylesheets/partials/_contacts.scss

```diff
@@ -0,0 +1,13 @@
+.contact {
+  &.index {
+    margin-top: 10px;
```

Comment:
Properties should be ordered border-bottom, margin-bottom, margin-top, padding-bottom